### PR TITLE
shell.fm: migrating to homebrew/boneyard

### PR DESCRIPTION
**Pull request by bfontaine**
Note that `shell.fm` no longer works because [last.fm stopped their radio streaming](http://nex.scrapping.cc/shell-fm/index.html).

### CODE REVIEW

**Comment by n054** [comment]
If the pushes came through an old device no longer in service would it help test to turn old on? Try to bounce of tower (self) build inside.

**Comment by n054** [comment]
Just a note, was using at a point an a FM Transmitter w/ 4(four) channels. This allowed m.device() to internet in a "cold" spot. (I have one on ().

File: Library/Formula/shell.fm.rb [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-  depends_on 'pkg-config' => :build
-  depends_on 'libao'
-  depends_on 'mad'
+  depends_on "pkg-config" => :build
+  depends_on "libao"
+  depends_on "mad"
 
   def install
+    # 'make install' fails if this exists
+    rm "INSTALL"
```
Comment (jacknagel) [inline Library/Formula/shell.fm.rb]: Upstream could address this by adding `.PHONY: install` to the Makefile.

**Comment by jacknagel** [comment]
Should we boneyard this, then?

**Comment by bfontaine** [comment]
@jacknagel Done: Homebrew/homebrew-boneyard#46

**Comment by tdsmith** [comment]
Thanks Baptiste!

**Pull request closed without merging**